katerynamelnykova/web-technologies-labs
# Added lab 6

# How to test this PR
To run the service:
```
uvicorn main:app --reload
```
To see the docs visit: http://localhost:8000/docs
